sweepbatcher/sweep_batcher_test.go

```diff
@@ -1029,3 +1030,83 @@ func TestGetFeePortionForSweep(t *testing.T) {
 		})
 	}
 }
+
+// TestRestoringEmptyBatch tests that the batcher can be restored with an empty
+// batch.
+func TestRestoringEmptyBatch(t *testing.T) {
+	defer test.Guard(t)()
+
+	lnd := test.NewMockLnd()
+	ctx, cancel := context.WithCancel(context.Background())
+
+	store := loopdb.NewStoreMock(t)
+
+	batcherStore := NewStoreMock()
+	_, err := batcherStore.InsertSweepBatch(ctx, &dbBatch{})
+	require.NoError(t, err)
+
+	batcher := NewBatcher(lnd.WalletKit, lnd.ChainNotifier, lnd.Signer,
+		testMuSig2SignSweep, nil, lnd.ChainParams, batcherStore, store)
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+
+	go func() {
+		defer wg.Done()
+		err = batcher.Run(ctx)
```

Comment:
Let's name it to `runErr` just in case another `err` is created between this line and the check of the error in the end.

Reply:
done